domains/certificates/chart/Chart.yaml

```diff
@@ -3,4 +3,4 @@ name: eo-certificates
 description: A chart containing the certificates domain.
 
 type: application
-version: 0.0.21
+version: 0.0.22
```

Comment:
I would argue these changes merit a major version change instead of a patch.

Reply:
bumped in 18708ff